docu/report/aufbau.tex

```diff
@@ -24,12 +60,36 @@ \subsection{Antenne}
+
 \subsection{Aufbau von LTE}
-\subsubsection{PSS}
-\subsubsection{SSS}
+Ein LTE-Signal besteht aus 504 unterschiedlichen Zellidentitäten auf physikalischer Ebene, diese sind in 168 unterschiedliche physikalischer Gruppen unterteilt. Eine Zellidentität setzt sich zusammen aus Identifikationsnummer der Gruppe sowie der Identifikationsnummer der jeweiligen physikalischen Ebene in der physikalischen Gruppe zusammen. Somit berechnet sich die Zellidentität wie folgt: $$N_{ID}^{cell}=3N_{ID}^{(1)}+N_{ID}^{(2)}$$ Hierbei gilt, dass jede Gruppe eine Nummer $N_{ID}^{(1)}$ zur Identifikation im Bereich von 0 bis 167 besitzt. Außerdem liegt die Identifikationsnummer $N_{ID}^{(2)}$ der physikalischen Ebene in der physikalischen Gruppe im Bereich von 0 bis 2.
+Zur Synchronisation des Signals benutzt LTE sowohl ein Primary Synchronisation Signal (PSS) als auch ein Secondary Synchronisation Signal (SSS).~\cite[S.~180]{etsi2021136}
+
+\subsubsection{Framestrukturtyp}
+Der Framestrukturtyp 1, der für Vollduplex und Halbdublex FDD gilt, hat eine Länge von 10 ms und besteht aus 10 Subframes, die jeweils eine Länge von $1ms$ haben und von 0 bis 9 durchnummeriert sind. Bei Subframes, die die Frequenz $\Delta f=2.5kHz$, $\Delta f=7.5kHz$ oder $\Delta f=15kHz$ besitzen wird ein Subframe in zwei Slots unterteilt, wo die länge jedes Slots $0.5ms$ beträgt. Bei einer Subframefrequenz von $\Delta f=1.25kHz$ besteht der Subframe aus keinen Slots und seine Länge beträgt $1ms$. Wenn die Übertragungsfrequenz bei $\Delta f=0.37kHz$ liegt, besitzt ein Slot eine Länge von $3ms$. Eine Periode ist $40ms$ lang und besteht dem nach aus 12 Slots, die von 0 bis 13 durchnummeriert sind. Die Übertragung beginnt somit bei jedem vierten Frame. Außerdem stehen für FDD 10 Subframes, 20 Slots oder bis 60 Subslots zur Verfügung.
```

Comment:
Halte die Sätze kurz. Du neigst dazu Sätze mit und aneinander zu reihen. Ein "und" sollte in einem Satz nur ein mal als Konjunktion genutzt werden. Allgemein solltest du dich auf einen Haupt- und einen Nebensatz beschränken. Das Erkennst du meistens an Schlüsselwörtern wie "sowie", "aber", "und", "als", "dass", "außer", "da".

Quantitäten wie Sekunden, Meter, Kilometer, Hertz, etc. kannst du am Besten mit dem package `siunitx` setzen. Das habe ich bisher in meinen Korrekturen ja schon vorgeführt.

Also statt:
```latex
$10ms$
```
besser:
```latex
\SI{10}{\milli\second}
```
Andere Beispiele sind `\SI{123}{\mega\hertz}` oder `\SI{300}{\metre\per\second}` (man beachte die französische Schreibweise, `\metre`, statt ~`\meter`~.
Details und Beispiele im [Manual](https://ctan.joethei.xyz/macros/latex/contrib/siunitx/siunitx.pdf).